frotz.py

```diff
@@ -125,6 +134,37 @@ def __init__(self, handle):
         # if read_file is called, that will override this
         GLib.idle_add(self.start_game, default_game_file)
 
+    def _create_view_toolbar(self):  # Zoom toolbar
+        view_toolbar = Gtk.Toolbar()
+        zoom_out_button = ToolButton('zoom-out')
+        zoom_out_button.set_tooltip(_('Zoom out'))
+        zoom_out_button.props.accelerator = '<Ctrl>minus'
+        zoom_out_button.connect('clicked', self.__zoom_out_cb)
+        view_toolbar.insert(zoom_out_button, -1)
+        zoom_out_button.show()
+
+        zoom_in_button = ToolButton('zoom-in')
+        zoom_in_button.set_tooltip(_('Zoom in'))
+        zoom_in_button.props.accelerator = '<Ctrl>plus'
+        zoom_in_button.connect('clicked', self.__zoom_in_cb)
+        view_toolbar.insert(zoom_in_button, -1)
+        zoom_in_button.show()
+        return view_toolbar
+
+    def _zoom(self, step):
+
```

Comment:
Why the empty line?

Reply:
Thanks now I have removed this commit. Please See https://github.com/sugarlabs/Frotz/pull/7#issuecomment-675881114